tests/unit/GoModal/GoModal.spec.js

```diff
@@ -0,0 +1,48 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import * as AgoUikit from '../../../src'
+import * as uiv from 'uiv'
+import { GoModal } from '../../../src'
+
+const localVue = createLocalVue()
+
+localVue.use(AgoUikit)
+localVue.use(uiv)
```

Comment:
I don't think you need to use uiv in localVue because AgoUikit already installing it.